torchaudio/datasets/iemocap.py

```diff
@@ -0,0 +1,140 @@
+import os
+import re
+from collections import defaultdict
+from pathlib import Path
+from typing import Optional, Tuple, Union
+
+from torch import Tensor
+from torch.utils.data import Dataset
+from torchaudio.datasets.utils import _load_waveform
+
+
+SAMPLE_RATE = 16000
+
+
+def _get_wavs_paths(data_dir):
+    wav_dir = os.path.join(data_dir, "sentences/wav")
+    wav_paths = sorted(str(p) for p in Path(wav_dir).glob("*/*.wav"))
+    relative_paths = []
+    for wav_path in wav_paths:
+        start = wav_path.find("Session")
+        wav_path = wav_path[start:]
+        relative_paths.append(wav_path)
+    return relative_paths
+
+
+class IEMOCAP(Dataset):
+    """*IEMOCAP* :cite:`iemocap` dataset.
+
+    Args:
+        root (str or Path): Root directory where the dataset's top level directory is found
+        sessions (Optional[List[int]], optional): List of sessions (1-5) to use. If ``None``, loads from all sessions.
```

Comment:
Didn't we standardize on the following format?

```suggestion
        sessions (List[int] or None, optional): List of sessions (1-5) to use. If ``None``, loads from all sessions.
```